scipy/stats/tests/test_kdeoth.py

```diff
@@ -390,3 +390,47 @@ def test_weights_integer():
     assert_allclose(pdf_i.evaluate(xn),
                     pdf_f.evaluate(xn), atol=1e-14, rtol=1e-14)
 
+
+def test_seed():
+    # Test the seed option of the resample method
+    def test_seed_sub(gkde_trail):
+        n_sample = 200
+        # The results should be different without using seed
+        samp1 = gkde_trail.resample(n_sample)
+        samp2 = gkde_trail.resample(n_sample)
+        assert_raises(
+            AssertionError, assert_array_almost_equal,
+            samp1, samp2, decimal = 13
```

Comment:
PEP8, remove the spaces: `decimal=13`

Reply:
In fact could you use `assert_allclose` here, with an `atol=1e-13`? If they're not supposed to be the same then perhaps the test tolerance can be relaxed to make the test harder?